recipes/xarray-extras/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: xarray-extras
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/x/xarray-extras/xarray_extras-{{ version }}.tar.gz
+  sha256: b73dc3ee40fdf38c5576383c40a5890d1010e346b72e8cf8f20cf7e8b66f19c8
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - dask
+    - numba
+    - numpy
+    - scipy
+    - xarray
```

Comment:
Please include the version constraints as well.

ref: https://github.com/crusaderky/xarray_extras/blob/v0.1.0/setup.py#L33-L34